jaknaeso-core/src/main/java/org/nexters/jaknaesocore/domain/character/model/ValueReports.java

```diff
@@ -0,0 +1,77 @@
+package org.nexters.jaknaesocore.domain.character.model;
+
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.Embeddable;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.nexters.jaknaesocore.domain.survey.model.Keyword;
+import org.nexters.jaknaesocore.domain.survey.model.KeywordScore;
+import org.nexters.jaknaesocore.domain.survey.model.SurveySubmission;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Embeddable
+public class ValueReports {
+
+  private static final BigDecimal PERCENTAGE100 = BigDecimal.valueOf(100);
+
+  @ElementCollection private List<ValueReport> reports;
+
+  private ValueReports(final List<ValueReport> reports) {
+    this.reports = reports;
+  }
```

Comment:
```suggestion
public class ValueReports {

  private static final BigDecimal PERCENTAGE100 = BigDecimal.valueOf(100);

  private List<ValueReport> reports;

  private ValueReports(final List<ValueReport> reports) {
    this.reports = reports;
  }
```
DB 엔티티 객체로 사용하는게 아니라서 Jpa관련 어노테이션은 제외해도 될 것 같은데 사용되는건가요 ??

Reply:
앗 일급 컬렉션 사용하면서 없애야 하는 코드였는데 그대로 두었네요! 수정하겠습니다